src/dcap_provider.cpp

```diff
@@ -86,6 +86,25 @@ void log(sgx_ql_log_level_t level, const char* fmt, ...)
     }
 }
 
+static std::string get_base_url() {
+    const char * env_base_url = getenv("AZURE_BASE_CERT_URL"); 
+    if (env_base_url == NULL)
+    {
+        return std::string(cert_base_url);
+    }
+    else
+    {
+        const size_t max_url_length = 2000; 

```

Comment:
We can probably lower this value, I think I chose 2000 because that was the standard max URL length for other projects I came across.

Reply:
nit: add #define MAX_URL_LENGTH 2000 at top of file, and reference it here.